boards/FSM/Src/Io/Io_FlowMeters.c

```diff
@@ -0,0 +1,82 @@
+#include "main.h"
+#include "Io_SharedFreqOnlyPwmInput.h"
+#include "Io_FlowMeters.h"
+#include <assert.h>
+
+struct FlowMeters_Context
+{
+    struct FreqOnlyPwmInput *pwm_input;
+    TIM_HandleTypeDef *      htim;
```

Comment:
htim is already in `struct FreqOnlyPwmInput`, so it seems redundant? Honestly I don't think this struct should exist at all because all the necessary detail should just belong in `struct FreqOnlyPwmInput`.

Reply:
Fixed with getter fxns https://github.com/UBCFormulaElectric/Consolidated-Firmware/pull/564#discussion_r411210500